backend/src/server.js

```diff
@@ -19,56 +24,56 @@ let db;
 app.use(bodyParser.json());
 app.use(express.static(path.join(__dirname, '/../../frontend/dist')));
 
-app.post('/api/items', async function (req, res) {
+app.post(api.items, async function (req, res) {
     let item = req.body;
     let validationResult = validation.checkItem(item);
     if (validationResult.isValid) {
         let added_id = await klad.placeInKladovka(db, Collections.Items, validationResult.item);
-        res.header('Location', '/api/items/' + added_id);
+        res.header('Location', api.items + added_id);
```

Comment:
This relies on the fact that api.items is ended with '/', I don't like it, it is errorprone - it is easy to forget that slash sometimes. May be use something like path.join to concatenate url and id? But I don't know if path.join is supposed to be used only with filepathes, please look it up.